packages/server/graphql/public/types/UpdateMeetingPromptSuccess.ts

```diff
@@ -0,0 +1,17 @@
+import {Maybe, ResolversTypes, UpdateMeetingPromptSuccessResolvers} from '../resolverTypes'
+
+export type UpdateMeetingPromptSuccessSource = {
+  meetingId: string
+}
+
+const UpdateMeetingPromptSuccess: UpdateMeetingPromptSuccessResolvers = {
+  meeting: async (source, _args, {dataLoader}) => {
+    const {meetingId} = source
+    const meeting = await dataLoader.get('newMeetings').load(meetingId)
+    return (meeting.meetingType !== 'teamPrompt' ? null : meeting) as Maybe<
```

Comment:
+1 no need to validate the meetingType here since `meetingId` is provided by the server (we already validated it in the resolve function of the mutation. can just return it as a `TeamPromptMeeting`. returning a nonnull for a success payload is always nice because then the client doesn't have to check to see if `meeting` exists.

my heuristic for these checks is asking myself if we could achieve 100% branch coverage if we wrote an integration test for it. if we can't test the branch that returns `null` then we don't need to build a case for it